frontend/src/Components/LoginForm.jsx

```
@@ -0,0 +1,73 @@
+  };
+
+  return (
+    <div className="login-container">
+      <div className="login-card">
+        <div className="login-header" style={{fontSize: '1.25rem'}}>
+          <p>Don't Have an account?<a href="/signup"><u>Sign up</u></a></p>
+        </div>
+        
+        <form onSubmit={handleSubmit}>
+          <div className="form-group">
+            <label>Username</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)